infrastructure/schedules.tf

```diff
@@ -111,4 +111,33 @@ resource "aws_lambda_permission" "statistical_report_schedule_permission" {
     module.statistical-report-lambda,
     aws_cloudwatch_event_rule.statistical_report_schedule
   ]
+}
+
+resource "aws_cloudwatch_event_rule" "update_patient_ods_schedule" {
```

Comment:
Maybe add a count on this, so it would only run on prod. I think we don't want this to run on dev every week.